README.md

```diff
@@ -464,6 +451,19 @@ the binary and [manpage](man/timg.1.md) on your system, type in the build direct
 sudo make install
 ```
 
+### macOS - Build and Install
+
+#### Install latest release
+```bash
+brew install timg
+```
+
+#### Build and install HEAD
```

Comment:
This would be the equivalent of pointing the users how to build the debian source package, but that is less useful for someone who actually wants to work with the code.

So I'd probably say something like: You can install timg directly from brew with `brew install timg`.
If you want to build from source .... (insert previous dependency install, cmake ...)